src/conduit/tests/test_check_db_migrated.py

```diff
@@ -0,0 +1,98 @@
+"""Test def check_db_migrated function."""
+
+from conduit import check_db_migrated
+from conduit import main
+from unittest import mock
+
+
+@mock.patch("conduit.alembic")
+def test_skip_when_running_an_alembic_command(alembic):
```

Comment:
Is this file intentionally not hinted?

Reply:
Hmmm. It is not. I wonder why mypy's coverage didn't report it.

Good catch!

Reply:
It's because `tests` folders don't have `__init__.py` files. Makes sense. Adding them now.